pulser-core/pulser/noise_model.py

```diff
@@ -287,6 +303,7 @@ def _find_relevant_params(
                 nt_ == "doppler"
                 or (nt_ == "amplitude" and amp_sigma != 0.0)
                 or (nt_ == "SPAM" and state_prep_error != 0.0)
+                or (nt_ == "detuning" and detuning_sigma != 0.0)
```

Comment:
`detuning` noise should not be active if `detuning_sigma` is 0, so you can forgo asking for the `detuning_sigma` value and just check if `detuning` is in `noise_types` (like is done for `doppler`)